src/vm/mlinfo.cpp

```diff
@@ -2134,7 +2133,7 @@ MarshalInfo::MarshalInfo(Module* pModule,
                             }
                             m_type = MARSHAL_TYPE_BSTR;
                             break;
-     
+#ifdef FEATURE_COMINTEROP     
```

Comment:
If we are already doing this work, why wouldn't we want to support also the NATIVE_TYPE_ANSIBSTR case ?
Is this due to lack of test coverage ?

Reply:
> Is this due to lack of test coverage ?

Yes. Currently we don't have test coverage for this scenario(Pinvoke ANSIBSTR). We are going to add more test coverage but it will take a while to add all of them.